modules/light-clients/08-wasm/types/contract_api.go

```diff
@@ -97,24 +97,11 @@ type checkSubstituteAndUpdateStateMsg struct{}
 
 // ContractResult defines the expected interface a Result returned by a contract call is expected to implement.
 type ContractResult interface {
-	Validate() bool
-	Error() string
 }
 
 // contractResult is the default implementation of the ContractResult interface and the default return type of any contract call
 // that does not require a custom return type.
 type contractResult struct {
-	IsValid  bool   `json:"is_valid,omitempty"`
-	ErrorMsg string `json:"error_msg,omitempty"`
-	Data     []byte `json:"data,omitempty"`
```

Comment:
None of these are used in the contracts at the moment. `IsValid` is always true and `ErrorMsg` and `Data` are always empty.